code/modules/projectiles/magazines/smgs.dm

```diff
@@ -117,3 +117,28 @@
 	max_rounds = 50
 	bonus_overlay = "uzi_ex"
 	icon_state_mini = "mag_smg_yellow"
+
+//-------------------------------------------------------
+//Vector, based on KRISS vector 45ACP.
+/obj/item/ammo_magazine/smg/vector
+	name = "\improper SMG-45 magazine (.45ACP)"
+	desc = "A .45ACP caliber magazine for the SMG-45."
+	default_ammo = /datum/ammo/bullet/smg/acp
+	caliber = CALIBER_45ACP
+	icon_state = "uzi_ext"
+	max_rounds = 25 // MagEx 25+ mag
+
+/obj/item/ammo_magazine/smg/vector/drum
```

Comment:
```suggestion
/obj/item/ammo_magazine/smg/vector/drum
```
Eh, we got rid of extended magazines for good reason, I don't see why we need these.

Reply:
there actually just repurposed ppsh mags by the looks of it so they are already in game dont know how that works for code though and honestly the reason for why drum mags got removed was both unknown to a lot of people and not really much of an issue anymore given how way xenos operate now (and with every smg in import basically being drum magged least we forget about the smg 25s 60 round mags :P)